flutter_vlc_player/lib/src/flutter_vlc_player.dart

```diff
@@ -47,9 +47,9 @@ class _VlcPlayerState extends State<VlcPlayer>
     };
   }
 
-  VoidCallback _listener;
+  late VoidCallback _listener;
 
-  bool _isInitialized;
+  bool? _isInitialized;
```

Comment:
Can be non-nullable:
```suggestion
  bool _isInitialized = false;
```

Reply:
resolved